udata/core/user/views.py

```diff
@@ -81,6 +82,22 @@ def get_context(self):
         return context
 
 
+    def get_json_ld(self, user=None):
+        if user is None:
+            user = self.user
+
+        result = {"@type": 'Person',
+                  "@context": "http://schema.org",
+                  "image": placeholder(user.avatar, 'user'),
```

Comment:
Placeholders are only for visual comfort and consistence. They should not be used when no image is available for JSON-LD. Simply output `image` only when it's present.